docker/ravendb-nanoserver/run-raven.ps1

```diff
@@ -2,7 +2,9 @@ $ErrorActionPreference = 'Stop'
 
 $COMMAND=".\Raven.Server.exe"
 $hostname = & "hostname.exe"
-$env:RAVEN_ServerUrl = "http://$($hostname):8080"
+if ([string]::IsNullOrEmpty($env:RAVEN_SETTINGS) -eq $True) {
```

Comment:
I think you wanted to say RAVEN_ServerUrl.

Reply:
indeed